builtin/providers/aws/resource_aws_appautoscaling_target.go

```diff
@@ -20,25 +20,6 @@ func resourceAwsAppautoscalingTarget() *schema.Resource {
 		Delete: resourceAwsAppautoscalingTargetDelete,
 
 		Schema: map[string]*schema.Schema{
-			"name": &schema.Schema{
```

Comment:
There is no `name` attribute for `aws_appautoscaling_target`. It's cut and paste from `aws_appautoscaling_policy`. The documentation even refers to *policy*. See commit message.